### ISSUE TITLE
Cutting off decimal zeroes

### ISSUE THREAD

**Issue by semaperepelitsa** [jakob/Postico#205]
When working with numeric fields it is hard to visually compare values because Postico cuts off decimal zeroes so digits don't line up vertically.

```sql
select * from (values ('100.000'::numeric(10,3)), ('101.020'), ('90.542'), ('95.920')) measures(t) order by t;
```

<img width="65" alt="screen shot 2016-04-15 at 15 56 07" src="https://cloud.githubusercontent.com/assets/347921/14561957/f95859c6-032a-11e6-9145-96a86e08f93f.png">

PSQL does it right:

```
    t    
---------
  90.542
  95.920
 100.000
 101.020
(4 rows)
```

**Comment by jakob** [comment]
Fixed. Will be released in next update.

You can try it right now in the latest dev build:
https://eggerapps-downloads.s3-eu-west-1.amazonaws.com/postico-1429.zip

**Comment by jakob** [comment]
Fixed in 1.0.7
